### PR TITLE
add dates

### PR DESCRIPTION
**Pull request by Vanshika-8**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,49 +4,49 @@
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <link rel="stylesheet" type="text/css" href="style.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.2/css/all.min.css" integrity="sha512-HK5fgLBL+xu6dm/Ii3z4xhlSUyZgTT9tuc/hSrtw6uzJOvgRr2a9jyxxT1ely+B+xFAmJKVSTbpM/CuL7qxO8w==" crossorigin="anonymous" />
```
Comment (claudialphonse) [inline index.html]: Don't use font awesome, I had given you a link for icons http://flaticon.com/
download icons from there and use them, you have to use images as icons

File: index.html [hunk 2]
```diff
@@ -4,49 +4,49 @@
...
+    
+    <ul class="list">
+      <li class="logo">
+        <i class="fas fa-list fa-2x"></i>
+        <div class="total-tasks">
+          Total Tasks: <span>0</span>
+        </div>
+      </li>
+      <li class="logo">
+        <i class="fas fa-search fa-2x"></i>
```
Comment (claudialphonse) [inline index.html]: you have to download and use the images

File: style.css [hunk 3]
```diff
@@ -18,21 +18,44 @@
...
+    width: 80%;
+    max-width: 400px;
+    height: 80%;
     display: flex;
     flex-direction: column;
-    align-items: center;
-    padding: 100px;
-    border: 50px;
+    text-align: center;
+    background-color: #F7F7F7;
```
Comment (claudialphonse) [inline style.css:46]: use CSS variables, look up and find what are CSS variables and use them

File: style.css [hunk 4]
```diff
@@ -109,11 +119,39 @@
...
+    color:#EF8768;
+  }
+  
+ .container #year{
+   color:lightgrey;
+ }
+  
+.container #day , #date{
+  font-size: 2rem;
+  float: left;
```
Comment (claudialphonse) [inline style.css]: you are already using flex then is there is a need for a float here?

File: todo.js [hunk 5]
```diff
@@ -7,17 +7,25 @@ const todoItemsList = document.querySelector('.todo-items')
 
 const totalTasks = document.querySelector(".total-tasks span")
 const completedTasks = document.querySelector(".completed-tasks span")
-const remainingTasks = document.querySelector(".remaining-tasks span")
+// const remainingTasks = document.querySelector(".remaining-tasks span")
```
Comment (claudialphonse) [inline todo.js]: remove console logs

File: todo.js [hunk 6]
```diff
@@ -7,17 +7,25 @@ const todoItemsList = document.querySelector('.todo-items')
...
 
 
+const currentDay=new Date()
```
Comment (claudialphonse) [inline todo.js]: rename variable to currentDate or dateObject or something as it denotes the complete date

File: todo.js [hunk 7]
```diff
@@ -7,17 +7,25 @@ const todoItemsList = document.querySelector('.todo-items')
...
+const showcurrentDay=currentDay.toLocaleString('default',{weekday:'long'})
+document.getElementById('day').innerHTML=showcurrentDay
+
+const currentDate=new Date()
```
Comment (claudialphonse) [inline todo.js]: no need to initialize this variable you already have a date object on top, reuse that variable to get the current date

File: todo.js [hunk 8]
```diff
@@ -7,17 +7,25 @@ const todoItemsList = document.querySelector('.todo-items')
...
+document.getElementById('date').innerHTML=currentDate.getDate()
+
+ const currentYear=new Date()
```
Comment (claudialphonse) [inline todo.js]: same here, if the variable holds the same value no need to add new variables try to reuse the data

File: todo.js [hunk 9]
```diff
@@ -7,17 +7,25 @@ const todoItemsList = document.querySelector('.todo-items')
...
+ document.getElementById('year').innerHTML=currentYear.getFullYear()
 
 
- 
 function countTasks() {
 totalTasks.textContent = todos.length
   const completedTasksArray = todos.filter((item) => item.completed)
   completedTasks.textContent = completedTasksArray.length
-  remainingTasks.textContent = todos.length - completedTasksArray.length
+  // remainingTasks.textContent = todos.length - completedTasksArray.length
```
Comment (claudialphonse) [inline todo.js]: remove console logs

**Review by claudialphonse — COMMENTED** [review]
View the review comments and make the necessary changes

File: index.html (hunk 1 above)
Comment (Vanshika-8) [reply index.html]: how to use that ?

Comment (claudialphonse) [reply index.html]: have you opened the link? you have to search the icon that you want and download it

Comment (Vanshika-8) [reply index.html]: yes , do i have to use the href tag for this?

Comment (claudialphonse) [reply index.html]: no, you have to download the image and use the image tag for this

File: todo.js [hunk 10]
```diff
@@ -3,29 +3,74 @@ let todos = []
...
+    searchBar.style.display='block'
+    todoForm.style.display='none'
+    todoItemsList.style.display='block'
+  }
+  
+}
+
+const todoList=document.querySelector('.todoList')
+todoList.addEventListener('click',function(e){
+    console.log(e)
```
Comment (claudialphonse) [inline todo.js]: remove console log

File: todo.js [hunk 11]
```diff
@@ -3,29 +3,74 @@ let todos = []
...
+document.getElementById('day').innerHTML=`${showcurrentDay},`
+document.getElementById('date').innerHTML=dataObject.getDate()
+document.getElementById('year').innerHTML=dataObject.toLocaleString('default',{ month:'long'})
 
-const currentDate=new Date()
-document.getElementById('date').innerHTML=currentDate.getDate()
-
- const currentYear=new Date()
- document.getElementById('year').innerHTML=currentYear.getFullYear()
+console.log(`${showcurrentDay}  ${dataObject.getDate()}` )
```
Comment (claudialphonse) [inline todo.js]: remove console logs

File: todo.js [hunk 12]
```diff
@@ -3,29 +3,74 @@ let todos = []
...
+
+
+
+
+function toggleList(listvalue){
+  if(listvalue=='task'){
+    searchBar.style.display='none'
+    todoForm.style.display='block'
+    todoItemsList.style.display='block'
+  }else if(listvalue=='cycle'){
```
Comment (claudialphonse) [inline todo.js]: list value =='search'
as it makes it easier to understand

File: todo.js [hunk 13]
```diff
@@ -3,29 +3,74 @@ let todos = []
...
+    console.log(e)
+  toggleList('task')
+  todoList.classList.add("active-color")
+  searchList.classList.remove("active-color")
+  searchList.classList.add("inactive-color")
+})
+
+const searchList=document.querySelector('.searchList')
+  searchList.addEventListener('click',function(e){
+    console.log(e)
```
Comment (claudialphonse) [inline todo.js]: remove console log

**Review by claudialphonse — CHANGES_REQUESTED** [review]

**Pull request merged**